manifest.toml

```diff
@@ -0,0 +1,98 @@
+packaging_format = 2
+
+id = "pleroma"
+name = "Pleroma"
+description.en = "Federated social networking server built on open protocols"
+description.fr = "Serveur de réseautage social fédéré basé sur des protocoles ouverts"
+
+version = "2.5.2~ynh1"
+
+maintainers = ["Anmol Sharma", "yalh76"]
+
+[upstream]
+license = "AGPL-3.0-only"
+website = "https://pleroma.social/"
+admindoc = "https://docs.pleroma.social/"
+code = "https://git.pleroma.social/pleroma/pleroma/"
+
+yunohost = ">= 11.1.19"
+architectures = "all"
+multi_instance = false
+ldap = true
+sso = true
+disk = "50M"
+ram.build = "50M"
+ram.runtime = "50M"
+
+[install]
+    [install.domain]
+    type = "domain"
+    full_domain = true
+
+    [install.init_main_permission]
+    type = "group"
+    default = "visitors"
+
+    [install.admin]
+    type = "user"
+
+    [install.password]
+    type = "password"
+
+    [install.name]
+    ask.en = "Choose a name for your Pleroma instance"
+    ask.fr = "Choisissez un nom pour votre instance Pleroma"
+    type = "string"
+    example = "My_Pleroma"
+    default = "My_Pleroma"
+
+    [install.registration]
+    ask.en = "Should registration be open to users who do not have a YunoHost account on the system?"
+    ask.fr = "L'inscription doit-elle être ouverte aux utilisateurs qui n'ont pas de compte YunoHost sur le système ?"
+    type = "boolean"
+    default = false
+
+    [install.cache]
+    ask.en = "Enable media-cache for your instance"
+    ask.fr = "Activer le cache média pour votre instance"
+    type = "boolean"
+    default = true
+
+    [install.size]
+    ask.en = "Select the cache size. If you did not enabled media-cache in above option then this option will have no effect"
+    ask.fr = "Sélectionner la taille du cache. Si vous n'avez pas activé le cache média ci-dessus, cette option n'aura aucun effet"
+    type = "string"
+    choices = ["2g", "5g", "10g", "20g", "40g", "80g"]
+    default = "5g"
+
+[resources]
+        [resources.sources.main]
+        arm64.url = "https://git.pleroma.social/pleroma/pleroma/-/jobs/234437/artifacts/download?file_type=archive"
+        arm64.sha256 = "2e2622a7a50284573fbb3dc66f0510c03d19403ab1e23d6b5dce7bddc11fca95"
+        amd64.url = "https://git.pleroma.social/pleroma/pleroma/-/jobs/234433/artifacts/download?file_type=archive"
+        amd64.sha256 = "8ef0bea62671d39e60f9e08d13109a4c332c552a1f855184063353987d46c84a"
+        armhf.url = "https://git.pleroma.social/pleroma/pleroma/-/jobs/234449/artifacts/download?file_type=archive"
+        armhf.sha256 = "abab2ad69704b999dbe1b995ce12aabc8069d9620ca332ecc621b20826d6db97"
```

Comment:
Not 100% sure how to test out configuring the autoupdate strategy, but if possible, can YunoHost use Pleroma's GitLab repository?